File: ejemplos-programacion/suma_condicional.pas
program SumaCondicional;
var
  a, b, suma: integer;
begin
  writeln('=== SUMA CON CONDICIONAL (Pascal) ===');
  write('Ingresa el primer numero: ');
  readln(a);
  write('Ingresa el segundo numero: ');
  readln(b);
  
  suma := a + b;
  writeln('La suma es: ', suma);
  
  { Condicional: si la suma es mayor a 100 }
  if suma > 100 then
    writeln('ALTO')
  else
    writeln('BAJO');
end.

File: ejemplos-programacion/suma_condicional.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b, suma;
    
    cout << "=== SUMA CON CONDICIONAL (C++) ===" << "\n";
    cout << "Ingresa el primer numero: ";
    cin >> a;
    cout << "Ingresa el segundo numero: ";
    cin >> b;
    
    suma = a + b;
    cout << "La suma es: " << suma << "\n";
    
    // Condicional: si la suma es mayor a 100
    if (suma > 100) {
        cout << "ALTO" << "\n";
    } else {
        cout << "BAJO" << "\n";
    }
    
    return 0;
}
